mirdata/orchset.py

```diff
@@ -50,26 +50,22 @@ def download(data_home=None, force_overwrite=False):
     save_path = utils.get_save_path(data_home)
     dataset_path = os.path.join(save_path, ORCHSET_DIR)
 
+    if exists(data_home) and not force_overwrite:
```

Comment:
`data_home` is the directory above the dataset

Reply:
this is actually intentional at the moment (`exists` builds the actual path). will change in future pr.